client/src/components/GeneralStyle.css

```diff
@@ -82,7 +82,7 @@
 
 .description {
   text-align: center;
-  font-size: 1.5rem;
+  font-size: 1.3rem;
```

Comment:
This has nothing to do with this line of the css but I'm still getting the issue of having to scroll down on the first screen to hit start.